src/app/(login)/sign/page.tsx

```
@@ -0,0 +1,147 @@
+    <div className="flex min-h-screen items-center justify-center bg-white px-6 py-10">
+      <div className="w-full max-w-sm bg-white p-6">
+        <div className="mb-6 flex items-center justify-between">
+          <h1 className="text-lg font-bold text-[#1E3A8A]">LG U+NIVERSE</h1>
+          <button type="button">
+            <X className="h-5 w-5 text-gray-500" />
+          </button>
+        </div>
+
+        <form className="space-y-4">
+          <div>
+            <label className="mb-1 block text-sm font-bold text-blue-950">이름</label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)